src/kibana4-dashboards/kibana4-dashboards.json.erb

```diff
@@ -0,0 +1,17 @@
+<%
+require 'json'
+require 'cgi'
+
+def include_template(type, name)
+    template_path = "src/kibana4-dashboards/#{type}/#{name}.json.erb"
+    "{\"index\":{\"_index\":\".kibana\",\"_type\":\"#{type}\",\"_id\":\"#{name}\"}}\n#{JSON.parse(ERB.new(File.read(template_path)).result(binding)).to_json}"
```

Comment:
Some more magic to convert each file into the

```
{"index":{"_index":".kibana","_id":"Overview-welcome","_type":"visualization"}}
{"title":"Overview welcome","visState":"{\"type\":\"markdown\", ... snip ...
```
pair wanted by Elasticsearch's `_bulk` API